lib/internal/bootstrap/node.js

```diff
@@ -521,6 +522,33 @@
     });
   }
 
+  function setupQueueMicrotask() {
+    const { queueMicrotask } = NativeModule.require('internal/queue_microtask');
+    Object.defineProperty(global, 'queueMicrotask', {
+      get: () => {
+        process.emitWarning('queueMicrotask() is experimental.',
+                            'ExperimentalWarning');
```

Comment:
What’s the motivation for making this experimental? I think I’m -1 on that… there is not much API surface here that we would need to adjust, right?

Reply:
people asked for it, I'm not really sure.

Reply:
Experimental because, AFAIK, there are no browser implementations of this yet so we cannot yet be certain that there is appropriate interop across implementations. Until we have at least one other implementation to verify against, we should keep it experimental. The issue here is not so much API surface area but timing concerns. Any time we have to change timing characteristics it tends to be semver-major, and given that this is being positioned as a potential replacement for `process.nextTick()` in many scenarios, it pays for us to take our time and be more cautious.

Reply:
The timing characteristics of this API are already exposed through the Promise API, right?

Lack of browser parity is definitely a valid argument… would browser adoption be a (sufficient) exit criterion for experimental status in that case?

Reply:
Likely yeah, but there also be some demonstrated use and a module in citgm that uses it

Reply:
it seems safari has implemented this in the technology preview

Reply:
I'm still -1 on removing experimental. It's clearly still juust being implemented by folks.